GUI/QtQuick/qml/Settings/DeleteArmoryServer.qml

```diff
@@ -88,7 +88,7 @@ ColumnLayout  {
 
             preferred: true
             function click_enter() {
-                armoryServersModel.del(server_index)
```

Comment:
Will this still delete a line from the model?

Reply:
the same with add